opalog/opalog.peg

```diff
@@ -69,10 +82,6 @@ Number <- '-'? Integer ( '.' DecimalDigit+ )? Exponent? {
     return t, err
```

Comment:
Sorry, the commit message was a bit confusing. The JSON spec says the limit is *defined by the implementation*. The spec recommends IEEE-754 double precision numbers (which is what float64 is in Go) for good interoperability. I agree that adding support for arbitrary precision is not needed right now.